uber/templates/base.html

```diff
@@ -175,19 +175,6 @@
     {% endif %}
     {% block top_of_body_additional %}{% endblock %}
     <div id="mainContainer" class="container-fluid">
-        {% block top %}
-            <div id="top" style="display:none">
-                <table style="width:100% ; background:orange ; margin-bottom:5px ; position:fixed">
-                <tr>
-                    <td style="width:auto"></td>
-                    <td style="width:30px">
-                        <a onClick="hideTop()" class="closer">x</a>
-                    </td>
-                </tr>
-                </table>
-                <br/><br/>
-            </div>
-        {% endblock %}
```

Comment:
This was our precursor to using ``toastr`` for info/error messages.  I went ahead and just switched our messages over to ``toastr`` since we're already using it pretty much everywhere else, which allowed me to delete this no-longer-used block.